src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1104,7 +1161,7 @@ resource pipeline_InitializeHub 'Microsoft.DataFactory/factories/pipelines@2018-
                       commandTimeout: '00:20:00'
                     }
                     linkedServiceName: {
-                      referenceName: dataset_dataExplorer.name
+                      referenceName: dataExplorerName
```

Comment:
```suggestion
                      referenceName: linkedService_dataExplorer.name
```

Reply:
Doesn't work if the resource is optional as the object will evaluate as null if we don't deploy ADX.  This causes some timing challenges too.